### ISSUE TITLE
Question about [training process of the prediction network]

### ISSUE THREAD

**Issue by auniquesun** [vita-epfl/monoloco#30]
When training your prediction network, where does the **ground truth distance _μ_ and spread _b_** come from?

In the Section 5 of the original paper, I am still unclear about how to obtain input-output pairs of 2D joints and distances, especially how to get the **ground truth distance _μ_ and spread _b_** of various kinds of predestrians.

It's really important to train a prediction network. I hope you can give more tips, explanations or references.

**Comment by bertoni9** [comment]
Hi, The ground-truth distance comes from KITTI labels as norm of the x,y,z vector (3D location of the person).  b is an unsupervised prediction of the network using the Laplace loss function (Eq.4). For more details you can also check _Alex Kendall and Yarin Gal. What uncertainties do we need in bayesian deep learning for computer vision?, NeurIPS _2017__

Best,
Lorenzo
